mxcube3/ui/components/SampleGrid/TaskButtons.jsx

```diff
@@ -4,28 +4,30 @@ import { Button, Glyphicon, ButtonToolbar } from "react-bootstrap"
 export default class SampleTaskButtons extends React.Component {
      constructor(props) {
          super(props);
-
-         this.showCharacterisationForm = this.props.showForm.bind(this, "Characterisation");
-         this.showDataCollectionForm = this.props.showForm.bind(this, "DataCollection");
+         this.showCharac = this.handleSubmit.bind(this, "Characterisation");
+         this.showDataCollec = this.handleSubmit.bind(this, "DataCollection");
```

Comment:
why using short names ? "Charac" or "Collec" doesn't mean anything, and it's only a few letters less than the normal word.